xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -451,7 +451,7 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct style>& subSty
           ConvColor(subStyle->shadowColor, subStyle->shadowOpacity); // Set the box shadow color
       style->Shadow = (10.00 / 100 * subStyle->shadowSize) * scale; // Set the box shadow size
       // By default a box overlaps the other, then we increase a bit the line spacing
-      lineSpacing = 8.0 * scaleDefault;
+      lineSpacing = fmax(subStyle->lineSpacing, 8.0 * scaleDefault);
```

Comment:
when you use c++ methods add namespace
```suggestion
      lineSpacing = std::fmax(subStyle->lineSpacing, 8.0 * scaleDefault);
```